packages/calcite-components/src/components/color-picker/color-picker.e2e.ts

```diff
@@ -912,18 +912,16 @@ describe("calcite-color-picker", () => {
     });
 
     it("value as property", async () => {
-      // initialize page with calcite-color-picker to make it available in the evaluate callback below
       const page = await newE2EPage({
-        html: "<calcite-color-picker></calcite-color-picker>",
+        html: "",
       });
-      await page.setContent("");
```

Comment:
This seems wrong.
The `html` option of newE2EPage just calls `setContent` behind the scenes.
Each time `setContent` is called, the page is re-created from scratch - there is no information carried over in between.

Reply:
**TL;DR**: it seems this workaround is no longer needed and we can simplify `newProgrammaticE2EPage`.

It looks like Stencil fixed an issue where creating an empty page and then adding a component via `createElement` would not load properly (i.e., the `html` content required a component). This was our workaround (basis for `newProgrammaticE2EPage`).